gossip-protocol-spec.md

```diff
@@ -121,21 +155,27 @@ A node sends it to ask the cluster for new information. It contains a bloom filt
 
 | Data | Type | Size | Description |
 |------|:----:|:----:|-------------|
-| `filter` | [`Bloom`](#bloom) | 24+ | a bloom filter |
-| `mask` | `u64` | 8 | filter mask |
-| `mask_bits` | `u32` | 4 | filter mask bits |
+| `filter` | [`Bloom`](#bloom) | 25+ | a bloom filter |
+| `mask` | `u64` | 8 | filter mask which defines the bloom filter |
```

Comment:
maybe can make a bit more clear: "defines the data stored in the bloom filter" or something. `mask_bits == 3` means that when creating the bloom filter, `CrdsValue` hashes that start with `000...x` go in filter 0, hashes that start with `001...x` go in filter 1, etc.